Dockerfile

```diff
@@ -1,28 +1,32 @@
-FROM ruby:2.7.2
+FROM ruby:2.7.6
 
-# throw errors if Gemfile has been modified since Gemfile.lock
+# Throw errors if Gemfile has been modified since Gemfile.lock
 RUN bundle config --global frozen 1
 
-RUN mkdir -p /usr/src/app
+# Set up the app directory
 WORKDIR /usr/src/app
 
-COPY Gemfile /usr/src/app/
-COPY Gemfile.lock /usr/src/app/
-RUN bundle install
+# Configure bundler
+ENV LANG=C.UTF-8 \
+    BUNDLE_JOBS=4 \
+    BUNDLE_RETRY=3
 
-COPY . /usr/src/app
+# Store Bundler settings in the project's root
+ENV BUNDLE_APP_CONFIG=.bundle
 
-EXPOSE 3000
+# Upgrade RubyGems and install the latest Bundler version
+RUN gem update --system && \
+    gem install bundler
+
+# Copy over the dependency files
+COPY Gemfile* .
 
-# install s6overlay so that we can run cron inside this container as well.
-ADD https://github.com/just-containers/s6-overlay/releases/download/v1.21.8.0/s6-overlay-amd64.tar.gz /tmp/
-RUN tar xzf /tmp/s6-overlay-amd64.tar.gz -C / \
-    && mkdir /config \
-    && apt-get update \
-    && apt-get install -y cron \
-    && rm -rf /var/lib/apt/lists/*
+# Install dependencies
+RUN bundle install
+
+# Copy over the rest of the app
+COPY . .
 
-COPY rootfs/ /
-VOLUME ["/config"]
+EXPOSE 3000
 
-ENTRYPOINT ["/init"]
+CMD ["bundle", "exec", "puma", "-C", "config/puma.rb"]
```

Comment:
@paigemoody this ended up being the way to invoke the app, rather than adding the `command` to `docker-compose`. Not sure if these are effectively the same.